nih-sparc/sparc-app
# Find data page - enhanced contentful fetching #44bvwu

# Description

The purpose of this PR is to enhance the contentful endpoint with the following:
- Move from `asyncData()` to a custom method
- Pagination, and offset functionality
- Dynamic search heading based on what is shown

## Type of change


# How Has This Been Tested?

- Visit the [Find Data page](http://localhost:3000/data?type=event)
- Events should load five at a time. On the first page, some events don't content. This is due to how they were entered in Contentful (some are drafts.) I will figure out the best way to do this down the road, but this is a unique situation for the **staging environment** and won't happen on production.
- Use pagination to see more results. Search Heading should update.
- Use the search form in the page's hero to search. Results should update.
- Click on one of the titles. You should be taken to that event's detail page.

# Checklist:
